client/gui/CIntObject.h

```diff
@@ -217,3 +215,13 @@ class WindowBase : public CIntObject
 protected:
 	void close();
 };
+
+class IStatusBar
+{
+public:
+	virtual ~IStatusBar();
+	virtual void clear() = 0;
+	virtual void clearMatching(const std::string & Text) = 0;
+	virtual void write(const std::string & Text) = 0;
+	virtual void lock(bool shouldLock) = 0;

```

Comment:
what is that for? Is it mutex lock? What shouldLock means? How behavior from other methods should differ if locked? For such interfaces it's good to have comments

Reply:
Strictly speaking, old code - this is part of CGStatusBar interface, but yeah, good point